.nvmrc

```diff
@@ -1 +1 @@
-18.12.0
+18.16.0
```

Comment:
please revert this, we need to use 18.12 as react and electron hasn't migrated to LTS